sirepo/package_data/static/json/impactt-schema.json

```diff
@@ -0,0 +1,206 @@
+            "y_offset": ["Misalignment Y [m]", "Float", 0],
+            "x_rotation": ["Rotation Error X [rad]", "Float", 0],
+            "y_rotation": ["Rotation Error Y [rad]", "Float", 0],
+            "z_rotation": ["Rotation Error Z [rad]", "Float", 0]
+        },
+        "DRIFTTUBE": {
+            "zedge": ["Z Edge", "Float", 0, "Longitudinal position of the entrance face of the element"]
+        },
+        "CONSTFOC": {
+            "zedge": ["Z Edge", "Float", 0, "Longitudinal position of the entrance face of the element"],
+            "radius": ["Radius [m]", "Float", 0],
+            "x_focusing_gradient": ["X Focusing Gradient [kx0^2]", "Float", 0],
+            "y_focusing_gradient": ["Y Focusing Gradient [ky0^2]", "Float", 0],
+            "z_focusing_gradient": ["Z Focusing Gradient [kz0^2]", "Float", 0]
+        },
+        "EMFLDCART": {
+            "zedge": ["Z Edge", "Float", 0, "Longitudinal position of the entrance face of the element"],
+            "rf_frequency": ["RF Frequency [Hz]", "RPNValue", 0, "Frequency of the RF field in Hz."],
+            "rf_field_scale": ["RF field scale", "Integer", 1, "This can be used to scale the field amplitude. Normally set to 1."],
+            "theta_0": ["Theta 0 [deg]", "RPNValue", 0, "Initial phase in degrees"],
+            "fmapfn": ["FMAPFN", "InputFile", "", "EMfldCart filename"]
+        },
+        "EMFLDCYL": {
+            "zedge": ["Z Edge", "Float", 0, "Longitudinal position of the entrance face of the element"],
+            "rf_frequency": ["RF Frequency [Hz]", "RPNValue", 0, "Frequency of the RF field in Hz."],
+            "radius": ["Radius [m]", "Float", 0],
+            "theta_0": ["Theta 0 [deg]", "RPNValue", 0, "Initial phase in degrees"],
+            "fmapfn": ["FMAPFN", "InputFile", "", "EMfldCyl filename"]
+        },
+        "EMFLDANA": {
+            "zedge": ["Z Edge", "Float", 0, "Longitudinal position of the entrance face of the element"],
+            "rf_frequency": ["RF Frequency [Hz]", "RPNValue", 0, "Frequency of the RF field in Hz."],
+            "e_scale": ["E Scale", "Float", 0],
+            "theta_0": ["Theta 0 [deg]", "RPNValue", 0, "Initial phase in degrees"],
+            "fmapfn": ["FMAPFN", "InputFile", "", "EMfldAna filename"]
+        },
+        "QUADRUPOLE": {
+            "_super": ["_", "model", "_ELEMENT"],
+            "b1_gradient": ["Gradient [T/m]", "Float", 0],
+            "L_effective": ["Effective Length [m]", "Float", 0]
+        },
+        "DIPOLE": {
+            "zedge": ["Z Edge", "Float", 0, "Longitudinal position of the entrance face of the element"],
+            "x_field_strength": ["X Field Strength [T]", "Float", 0],
+            "y_field_strength": ["Y Field Strength [T]", "Float", 0],
+            "half_gap_width": ["Half of Gap Width (m)", "Float", 0],
+            "fmapfn": ["FMAPFN", "InputFile", "", "file ID to contain the geometry information of bend."]
+        },
```

Comment:
DIPOLE also has misalignments and rotations. 
We want to match the lume-impact naming conventions as much as possible, so the field names should be (b_field_x, b_field, filename, and half_gap)
Try the lume-impact parser to see, or refer to their parser implementation:
https://github.com/ChristopherMayes/lume-impact/blob/master/impact/parsers.py

Reply:
on page 13 of the [impact manual](https://portal.nersc.gov/project/m669/IMPACT-T/documents/ImpactTv2.pdf) it says misalignment and rotation is not used for dipole. 

It also looks like in lume-impact's `parse_dipole(line)` those values dont get parsed. 

I will work on the name consistency with lume-impact